xml/telnet_banners.xml

```diff
@@ -816,4 +816,589 @@
+    <param pos="0" name="hw.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Linux"/>
+    <param pos="0" name="hw.device" value="General"/>
+    <param pos="1" name="os.version"/>
+  </fingerprint>
+  <fingerprint pattern="Check Point FireWall-1 authenticated Telnet server running on (.*)">
+    <description>Check Point Firewall-1</description>
+    <example _encoding="base64" host.name="gaatdrf2">
+      Q2hlY2sgUG9pbnQgRmlyZVdhbGwtMSBhdXRoZW50aWNhdGVkIFRlbG5ldCBzZXJ2ZXIgcnVubmluZyBvbiBnYWF0ZHJmMgoKVXNlcjog
+    </example>
+    <param pos="0" name="hw.vendor" value="Checkpoint"/>
+    <param pos="0" name="hw.family" value="Checkpoint"/>
+    <param pos="0" name="hw.device" value="Firewall"/>
+    <param pos="0" name="hw.product" value="Checkpoint FW1"/>
+    <param pos="1" name="host.name"/>
+  </fingerprint>
+  <fingerprint pattern="Raptor Firewall">
+    <description>Raptor Firewall</description>
+    <example _encoding="base64">
+      UmFwdG9yIEZpcmV3YWxsIFNlY3VyZSBHYXRld2F5LgoKSG9zdG5hbWU6IA==
+    </example>
+    <param pos="0" name="hw.vendor" value="Symantec"/>
+    <param pos="0" name="hw.family" value="Raptor"/>
+    <param pos="0" name="hw.device" value="Firewall"/>
+    <param pos="0" name="hw.product" value="Raptor"/>
+  </fingerprint>
+  <fingerprint pattern="UNIX\(r\) System V Release 4.0">
+    <description>System is SunOS 5.x (Solaris)</description>
+    <example _encoding="base64">
+      VU5JWChyKSBTeXN0ZW0gViBSZWxlYXNlIDQuMCAoVGhlLVNlcnZlcikKCgoKbG9naW46IA==
+    </example>
+    <param pos="0" name="hw.vendor" value="Sun"/>
+    <param pos="0" name="hw.family" value="Solaris"/>
+    <param pos="0" name="hw.device" value="General"/>
+    <param pos="0" name="hw.product" value="Solaris"/>
+  </fingerprint>
+  <fingerprint pattern="Solaris (.*)">
+    <description>System is Solaris</description>
+    <example _encoding="base64" os.version="1.1.1.B">
+      U2VhdHRsZSBDb21tdW5pdHkgTmV0d29yayBTdW4gU29sYXJpcyAxLjEuMS5CClBsZWFzZSBsb2dpbiBhcyAndml
+      zaXRvcicgaWYgeW91IGFyZSBhIHZpc2l0b3IKCgpTdW5PUyBVTklYIChzY24pCgoKCmxvZ2luOg==
+    </example>
+    <param pos="0" name="hw.vendor" value="Sun"/>
+    <param pos="0" name="hw.family" value="Solaris"/>
+    <param pos="0" name="hw.device" value="General"/>
+    <param pos="0" name="hw.product" value="Solaris"/>
+    <param pos="1" name="os.version"/>
+  </fingerprint>
+  <fingerprint pattern="Digital UNIX">
+    <description>System is Digital Unix</description>
+    <example _encoding="base64">
+      RGlnaXRhbCBVTklYIChqb3VybmFsKSAodHR5cDIpCgoKCmxvZ2luOiA=
+    </example>
+    <param pos="0" name="hw.vendor" value="HP"/>
+    <param pos="0" name="hw.family" value="Digital Unix"/>
+    <param pos="0" name="hw.device" value="General"/>
+    <param pos="0" name="hw.product" value="Digital Unix"/>
+  </fingerprint>
+  <fingerprint pattern="Compaq Tru64 UNIX V(.*) \(Rev. (.*\d)\) .*" flags="REG_MULTILINE">
+    <description>System is Compaq Tru64 UNIX V</description>
```

Comment:
Here and in the other descriptions that start with "System is", remove that snippet.